### PR TITLE
New blog post about 2.5.0 release

### PR DESCRIPTION
**Pull request by aromanenko-dev**

### CODE REVIEW

File: src/_posts/2018-06-22-beam-2.5.0.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+* UDAFs can be indirect suclasses of CombineFn
+* Many other small bugfixes
+
+## Portability
+* Common shared code related to supporting portable execution for runners.
+* Python SDK supporting side inputs over the portability APIs.
+
+
+## Extract metrics in a runner agnostic way
+Metrics are pushed by the runners to configurable sinks (Http REST sink available). It is already enabled in Filnk and Spark runner, work is in progress for Dataflow.
```
Comment (andrewrjones) [inline src/_posts/2018-06-22-beam-2.5.0.md]: Typo: Filnk -> Flink

File: src/_posts/2018-06-22-beam-2.5.0.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+---
+
+We are glad to present the new 2.5.0 release of Beam. This release includes
+multiple fixes and new functionalities. <!--more--> For more information
+please check the detailed release notes.
+
+# New Features / Improvements
+
+## Go SDK support
+The Go SDK has been officially accepted into the project, after an incubation period and community effort. Go pipelines run on Dataflow and all portable runners, notably the Flink runner. More details are [here(https://beam.apache.org/documentation/sdks/go/)].
```
Comment (tweise) [inline src/_posts/2018-06-22-beam-2.5.0.md]: Portable Flink runner isn't part of the release and not ready yet, so please remove this reference.

File: src/_posts/2018-06-22-beam-2.5.0.md (hunk 1 above)
Comment (aromanenko-dev) [reply src/_posts/2018-06-22-beam-2.5.0.md]: Fixed

File: src/_posts/2018-06-22-beam-2.5.0.md (hunk 2 above)
Comment (aromanenko-dev) [reply src/_posts/2018-06-22-beam-2.5.0.md]: Fixed

**Comment by jbonofre** [comment]
@asfgit merge

**Pull request closed without merging**